Framework/src/QcInfoLogger.cxx

```diff
@@ -59,6 +60,13 @@ void QcInfoLogger::setPartition(const std::string& partitionName)
   ILOG(Debug, Devel) << "IL: Partition set to " << partitionName << ENDM;
 }
 
+void QcInfoLogger::disable(bool disabled)
+{
+  QcInfoLogger::disabled = disabled;
```

Comment:
Since the method lets you also enable IL by setting `disabled` to `false`, the filtering settings should come back to the selected values in the config file.

Alternatively, you could make the `disable()` allow only to disable, but not the other way.

Reply:
Yes, let's not aim for the moon, I will only let the disabling :-D